bestofjs/bestofjs
# Refactor backend tasks and split schema

## Goal

- Split the big shema file into separate files (basically one by table, excluding relation tables)
- Simplify tasks, using where clauses introducing here #331  instead of `shouldProcess` functions to filter the projects to process

Example: `{ where: notInArray(schema.projects.status, ["deprecated", "hidden"]) }`

## How to test

Ensure tasks such as `build-static-api` only process projects that are not `deprecated`.

```sh
 bun run apps/backend/src/cli.ts build-static-api 
```
